src/Symfony/Component/Notifier/Bridge/Prelude/PreludeOptions.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Prelude;
+
+use Symfony\Component\Notifier\Message\MessageOptionsInterface;
+
+/**
+ * @author Imad Zairig <imadzairig@gmail.com>
+ */
+final class PreludeOptions implements MessageOptionsInterface
+{
+    public function __construct(
+        private readonly ?string $templateId = null,
+        private readonly array $variables = [],
+        private readonly ?string $from = null,
+        private readonly ?string $locale = null,
+        private readonly ?string $expiresAt = null,
+        private readonly ?string $scheduleAt = null,
+        private readonly ?string $callbackUrl = null,
+        private readonly ?string $correlationId = null,
+        private readonly ?string $preferredChannel = null,
```

Comment:
We have quite some inconsistency in the way the various bridges define their options class.
The proposed one is very unlike the majority (eg things started with an $options argument and setters/getter without set/get prefixes.)
It's a bit inconsistent already, but let's define how we want to improve consistency? /cc @OskarStark